nbgrader/apps/releasefeedbackapp.py

```diff
@@ -0,0 +1,81 @@
+# coding: utf-8
+
+from traitlets import default
+
+from .baseapp import NbGrader, nbgrader_aliases, nbgrader_flags
+from ..exchange import Exchange, ExchangeReleaseFeedback, ExchangeError
+
+
+aliases = {}
+aliases.update(nbgrader_aliases)
+aliases.update({
+    "timezone": "Exchange.timezone",
+    "course": "Exchange.course_id",
+})
+
+flags = {}
+flags.update(nbgrader_flags)
+
+class ReleaseFeedbackApp(NbGrader):
+
+    name = u'nbgrader-release-feedback'
+    description = u'Release assignment feedback to the nbgrader feedback exchange'
+
+    aliases = aliases
+    flags = flags
+
+    examples = """
+        Release feedback for an assignment to students. For the usage of instructors.
+
+        This command is run from the top-level nbgrader folder. 
+
+        The command releases the feedback present in the `feedback` folder. To populate 
+        tis folder use the `nbgrader feedback` command. 
+
+        To release the feedback for an assignment named `assignment1` run:
+
+            nbgrader release_feedback assignment1
+        
+        Release feedback overrides existing files. It should not be a problem given
+        that the feedback is associated with the hash of hte notebook. Any new notebook
+        will map to a different file. The only way a file actually gets replaced is when
+        the same input notebook gets re-graded and in these cases one would want the latest
+        grading to be the right one.   
+
+        """
+    @default("classes")
+    def _classes_default(self):
+        classes = super(ReleaseFeedbackApp, self)._classes_default()
+        classes.extend([Exchange, ExchangeReleaseFeedback])
+        return classes
+
+    def _load_config(self, cfg, **kwargs):
+        if 'ReleaseFeedbackApp' in cfg:
```

Comment:
Same thing here, I don't think you need this warning.

Reply:
Removed method in https://github.com/danielmaitre/nbgrader/pull/8